### ISSUE TITLE
Use quotes when resolving to support case-sensitive column names

### ISSUE THREAD

**Issue by 0x7061** [Antman261/tiny-fixtures#3]
**Is your feature request related to a problem? Please describe.**
When column names are in camel case e.g. `refreshToken` the current implementation will pass them as is, and postgres/pg will interpret them in lowercase format. This results into column not found issues where in this example it will try to insert into `refreshtoken`.

**Describe the solution you'd like**
In Postgres we can use quotes to keep case-sensitive column names. For the implementation this would mean a small change in the `query.ts` file:
```
const buildInsertColumnString = (row: Row) =>
  `(${Object
    .keys(row)
    .map((e) => `"${e}"`) // <-- apply quotes to each column name
    .join(', ')})`;
```

**Comment by Antman261** [comment]
Great suggestion, thank you! I hope to make some improvements to the package this weekend, sorry for the delay 😅

**Comment by Antman261** [comment]
Resolved in v0.2.5
